_posts/2015-11-09-scrum-basics-2.md

```diff
@@ -0,0 +1,118 @@
+Coaching the Product Owner is another big topic.
+As you have already learned, the PO role is quite complex and there are several techniques that can help him in his daily business.
+But someone needs to teach him those techniques and also make him aware of problems.
+Only if someone tells the PO that something is not working, they can improve themselves and their work.
+The same goes for the Development Team.
+It’s the SM's job to constantly push them to improve themselves, to become more self-organised and to be open for changes.
+
+You may have noticed that I switched from the first to the third biggest part of the Scrum Master's job, so here is the second: They have to ensure that impediments are removed.
+Impediments are all things that block the team from performing.
+If it is the manager in the room, a computer that’s too slow or a suddenly unusual high amount of support tickets: The Scrum Master needs to investigate and find a solution.
+That doesn’t mean that they have to solve everything by themselves.
+They only need to find the responsible people and ensure that they are solving the problem.
+Like pushing the decision maker for a fast “go” on a new computer to replace the slow one.
+To order and install it is then up to IT.
+Or the SM would have to investigate why there are suddenly so many support tickets.
+Has the quality dropped?
+Or is there a high sick-leave rate in the Service Center so they aren’t able to work in detail on every ticket and just pass them on?
+The SM would need to talk to them to know, if the Development Team might plan less tickets in the next sprint in order to have more support time.
+
+Two more tasks are left for the Scrum Master to do: facilitating meeting and mediation talks.
+The SM is responsible to moderate all the Scrum meetings and to make sure the meetings start and end on time.
+But if they do their job correctly and improved the team's self-organisation, they will soon take over moderation, e.g. for the review.
+Equally important are mediations skills.
+If there are problems within a team or also between teams the Scrum Master has to solve them.
+The SM needs to be the person of trust for every team member.
+
+In total you could say the Scrum Master has to keep the process running.
+No matter what happens, he has to make sure the Scrum Team is still able to work on their tickets.
+For that, it’s also important to promote Scrum in the organisation.
+If for example the Marketing department knows that the Scrum Team shows their development results every two weeks, they might just join these meetings to be up to date all the time.
+If managers know the two-week time frames, they can go early to the POs and ask if there's space for an additional task in the next sprint.
+Combine “protect the team”, “remove impediments”, “push for constant improvement” and “moderate meeting so they have a valuable outcome and end in time” and you can see where the value of this extra employee pushes in.
```

Comment:
Combine “protect the team”, “remove impediments”, “push for constant improvement” and “moderate meeting so they have a valuable outcome and end on time” and you can see where the value of this extra employee pushes in.